File: new-algorithm/src/javaalgo/implementation/boj2438.java
package javaalgo.implementation;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class boj2438 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int N = Integer.parseInt(br.readLine());

        for (int i = 1; i <= N; i++) {
            for (int j = N - i; j < N; j++) {
                System.out.print("*");
            }
            System.out.println();
        }
    }
}

File: new-algorithm/src/kotlinalgo/implementation/boj2438.kt
package kotlinalgo.implementation

import java.io.BufferedReader
import java.io.InputStreamReader

fun main() = with(BufferedReader(InputStreamReader(System.`in`))) {
    val N = Integer.parseInt(readLine())

    for (i in 1..N) {
        for (j in N - i until N) {
            print("*")
        }
        println()
    }
}